officemix/static/js/officemix.js

```diff
@@ -0,0 +1,58 @@
+function OfficeMixBlock(runtime, element) {
+  var iframe = $('iframe', element);
+  var player = new playerjs.Player(iframe.get(0));
+
+  player.on('ready', function () {
+      console.log('loaded');
+
+      player.getDuration(function (duration) {
+          console.log('duration:', duration);
+          var handlerUrl = runtime.handlerUrl(element, 'publish_event');
+          var data = {
+              'event_type': 'xblock.officemix.loaded',
+              'video': iframe.get(0).src,
```

Comment:
prefer iframe.attr('src'), but use a variable out of this scope so it’s not computed a thousand times